packages/hash/api/src/auth/ory-kratos.ts

```diff
@@ -15,9 +15,25 @@ export const adminKratosSdk = new OpenSourceV0alpha2Api(
 );
 
 export type KratosUserIdentityTraits = {
+  shortname?: string;
   emails: string[];
 };
 
 export type KratosUserIdentity = Omit<Identity, "traits"> & {
   traits: KratosUserIdentityTraits;
 };
+
+export const createKratosIdentity = async (params: {
```

Comment:
@thehabbos007 I'm going to hold of merging this PR until https://github.com/hashintel/hash/pull/947#pullrequestreview-1076151405 is in, so that I can refactor your method slightly to make use of this one.

Reply:
Have done this in https://github.com/hashintel/hash/pull/950/commits/b073ac96f98ed7b3718abaa82bc72713c7fc0c2c